src/dotnet/UI.Blazor.App/Pages/Landing/landing.css

```diff
@@ -0,0 +1,171 @@
+.landing {
+    @apply w-full;
+}
+.landing .page {
+    @apply flex-y h-screen overflow-hidden;
+}
+.landing .page-scrolling {
+    @apply flex-y h-full;
+}
+.landing .page-1 {
+    @apply bg-[#0436A3];
+}
+.landing .page-1 .page-1-wrapper {
+    @apply w-full h-full pt-16 bg-contain bg-no-repeat;
+    @apply bg-[url('../../../../dotnet/App.Wasm/wwwroot/landing/ellipses.svg')];
```

Comment:
Will this url value  be transformed? This css file will be transformed to '/dist/bundle.css/'. So I suppose that relatvie url should be set against path '/dist/bundle.css/.
I think it's incorrect that if we have 'dotnet/App.Wasm/wwwroot' in final style.